etcdctl/ctlv3/command/printer_simple.go

```diff
@@ -180,6 +182,14 @@ func (s *simplePrinter) RoleAdd(role string, r v3.AuthRoleAddResponse) {
 
 func (s *simplePrinter) RoleGet(role string, r v3.AuthRoleGetResponse) {
 	fmt.Printf("Role %s\n", role)
+	if rootRole == role {
```

Comment:
Can we make printRange return a better formatting of "".."\x00" range instead of a dedicated case for root ?

Reply:
You means that we didn't use `<open ended>` to take place "\x00", println "\x00" directly?

Reply:
I mean integrate it as an 'if' in the generic formatter: 

https://github.com/etcd-io/etcd/pull/12979/files/dbea955e33de552b468ffaf72583ce9064925c59#diff-90edaa0bc3c5019cf9efc36c455900a32b0e35c96fff18364e95cbc7a1bc61e0R198-R202

Reply:
It maybe not works, If r.Perm is empty, the result will not be print.

Reply:
I would go for something like: 
```
if (role==rootRole && r.Perm=nil) {
    ... 
    fmt.Printf("\t[, <open ended>")
}
```

- this uses the same format as we use for regular roles
- this will work even after we will do server side change to return "".."\x00" for root roles.

Reply:
fine.